diff of execution of:
echo 'export A------=a 123=4 B=b MINUS-' | tr ';' '\n' | [shell]
--- expected	2024-09-01 20:04:09.780678560 +0200
+++ found	2024-09-01 20:04:09.856678633 +0200
@@ -1,7 +1,5 @@
 
 ======== stderr =======
 [progname]: export: `A------=a': not a valid identifier
-[progname]: export: `123=4': not a valid identifier
-[progname]: export: `MINUS-': not a valid identifier
 ======== return =======
 return value: 1
