diff of execution of:
echo 'rm *; *' | tr ';' '\n' | [shell]
--- expected	2024-09-11 23:42:40.437029370 +0200
+++ found	2024-09-11 23:42:40.532026982 +0200
@@ -1,5 +1,8 @@
 
 ======== stderr =======
-[progname]: *: command not found
+[progname]: cannot remove '.': Is a directory
+[progname]: cannot remove '..': Is a directory
+[progname]: .: filename argument required
+[progname]: usage: . filename [arguments]
 ======== return =======
 return value: 1
