diff of execution of:
echo 'rm *; *' | tr ';' '\n' | [shell]
--- expected	2024-09-10 09:26:57.883582169 +0200
+++ found	2024-09-10 09:26:57.905582168 +0200
@@ -1,5 +1,7 @@
 
 ======== stderr =======
-[progname]: *: command not found
+[progname]: cannot remove '..': Is a directory
+[progname]: cannot remove '.': Is a directory
+[progname]: ..: command not found
 ======== return =======
 return value: 1
