diff of execution of:
echo 'export' | tr ';' '\n' | [shell]
--- expected	2024-08-29 22:59:07
+++ found	2024-08-29 22:59:07
@@ -1,40 +1,40 @@
-declare -x COLORTERM="truecolor"
-declare -x COMMAND_MODE="unix2003"
-declare -x GIT_ASKPASS="/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh"
-declare -x HOME="/Users/bruno"
-declare -x HOMEBREW_CELLAR="/opt/homebrew/Cellar"
-declare -x HOMEBREW_PREFIX="/opt/homebrew"
-declare -x HOMEBREW_REPOSITORY="/opt/homebrew"
-declare -x INFOPATH="/opt/homebrew/share/info:/opt/homebrew/share/info:"
-declare -x LANG="es_ES.UTF-8"
-declare -x LOGNAME="bruno"
-declare -x LaunchInstanceID="CA304FFA-E941-471B-B80F-EEC62BDE3794"
-declare -x MallocNanoZone="0"
-declare -x OLDPWD="/Users/bruno/Github_42/Minishell/test"
-declare -x ORIGINAL_XDG_CURRENT_DESKTOP="undefined"
-declare -x PATH="/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.12/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.12/bin"
-declare -x PWD="/Users/bruno/Github_42/Minishell/test/temp"
-declare -x SECURITYSESSIONID="186b2"
-declare -x SHELL="/bin/zsh"
-declare -x SSH_AUTH_SOCK="/private/tmp/com.apple.launchd.hcBou38jjP/Listeners"
-declare -x TERM="xterm-256color"
-declare -x TERM_PROGRAM="vscode"
-declare -x TERM_PROGRAM_VERSION="1.92.2"
-declare -x TMPDIR="/var/folders/1k/snmc95211t7fx2g2fgf6yp880000gn/T/"
-declare -x USER="bruno"
-declare -x USER_ZDOTDIR="/Users/bruno"
-declare -x VSCODE_GIT_ASKPASS_EXTRA_ARGS=""
-declare -x VSCODE_GIT_ASKPASS_MAIN="/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js"
-declare -x VSCODE_GIT_ASKPASS_NODE="/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Plugin).app/Contents/MacOS/Code Helper (Plugin)"
-declare -x VSCODE_GIT_IPC_HANDLE="/var/folders/1k/snmc95211t7fx2g2fgf6yp880000gn/T/vscode-git-57e233b12e.sock"
-declare -x VSCODE_INJECTION="1"
-declare -x XPC_FLAGS="0x0"
-declare -x XPC_SERVICE_NAME="0"
-declare -x ZDOTDIR="/Users/bruno"
-declare -x __CFBundleIdentifier="com.microsoft.VSCode"
-declare -x __CF_USER_TEXT_ENCODING="0x1F5:0x0:0x8"
+declare -x TERM_PROGRAM=vscode
+declare -x TERM=xterm-256color
+declare -x SHELL=/bin/zsh
+declare -x HOMEBREW_REPOSITORY=/opt/homebrew
+declare -x TMPDIR=/var/folders/1k/snmc95211t7fx2g2fgf6yp880000gn/T/
+declare -x TERM_PROGRAM_VERSION=1.92.2
+declare -x ZDOTDIR=/Users/bruno
+declare -x ORIGINAL_XDG_CURRENT_DESKTOP=undefined
+declare -x MallocNanoZone=0
+declare -x USER=bruno
+declare -x COMMAND_MODE=unix2003
+declare -x SSH_AUTH_SOCK=/private/tmp/com.apple.launchd.hcBou38jjP/Listeners
+declare -x __CF_USER_TEXT_ENCODING=0x1F5:0x0:0x8
+declare -x PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.12/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin:/opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.12/bin
+declare -x LaunchInstanceID=CA304FFA-E941-471B-B80F-EEC62BDE3794
+declare -x USER_ZDOTDIR=/Users/bruno
+declare -x __CFBundleIdentifier=com.microsoft.VSCode
+declare -x PWD=/Users/bruno/Github_42/Minishell/test/temp
+declare -x LANG=es_ES.UTF-8
+declare -x VSCODE_GIT_ASKPASS_EXTRA_ARGS=
+declare -x XPC_FLAGS=0x0
+declare -x XPC_SERVICE_NAME=0
+declare -x VSCODE_INJECTION=1
+declare -x HOME=/Users/bruno
+declare -x VSCODE_GIT_ASKPASS_MAIN=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass-main.js
+declare -x HOMEBREW_PREFIX=/opt/homebrew
+declare -x LOGNAME=bruno
+declare -x VSCODE_GIT_IPC_HANDLE=/var/folders/1k/snmc95211t7fx2g2fgf6yp880000gn/T/vscode-git-57e233b12e.sock
+declare -x VSCODE_GIT_ASKPASS_NODE=/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper (Plugin).app/Contents/MacOS/Code Helper (Plugin)
+declare -x GIT_ASKPASS=/Applications/Visual Studio Code.app/Contents/Resources/app/extensions/git/dist/askpass.sh
+declare -x INFOPATH=/opt/homebrew/share/info:/opt/homebrew/share/info:
+declare -x HOMEBREW_CELLAR=/opt/homebrew/Cellar
+declare -x SECURITYSESSIONID=186b2
+declare -x COLORTERM=truecolor
+declare -x OLDPWD=/Users/bruno/Github_42/Minishell/test
 
 ======== stderr =======
 
 ======== return =======
 return value: 0
